components/form-builder/app/edit/SelectedElement.tsx

```diff
@@ -65,7 +65,7 @@ export const SelectedElement = ({
       }
       break;
     case "email":
-      element = <ShortAnswer data-testid="email">name@example.com</ShortAnswer>;
```

Comment:
For this one example.com is pretty standard when using an example domain --- if you visit the website it actually states it purpose vs company.com

https://example.com

https://www.company.com

Reply:
Ok, can switch it back - was trying to make it less redundant, but no worries if it's a standard. Will just focus on making the French one French. 😅 Thanks Tim!